### PR TITLE
[feature] Button component added

### PR DESCRIPTION
**Pull request by KonradKlimczak**

### CODE REVIEW

File: src/styles/notification.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .notification {
   position: fixed;
   z-index: 1;
-  background: $primary-color;
+  background: blue;
```
Comment (enril) [inline src/styles/notification.scss]: Maybe using variables is better idea, than just word like blue.

File: test/__snapshots__/Button.test.tsx.snap [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (enril) [inline test/__snapshots__/Button.test.tsx.snap]: No idea... Could you tell me the difference between Button.test.tsx vs Button.test.tsx.snap

**Comment by enril** [comment]
Oh man, this should be 2 or 3 PR. 
1) Add Travis
2) Update libs - plus refactor
3) Add Button 👍 + test

https://github.com/KonradKlimczak/mikskarpety/pull/2/commits/77b941c2b959ffb23b325519ba9164a976dc76e7

File: test/__snapshots__/Button.test.tsx.snap (hunk 2 above)
Comment (KonradKlimczak) [reply test/__snapshots__/Button.test.tsx.snap]: It's just a spanshot generated by jest. It's just to catch unexpected DOM changes.

File: src/styles/notification.scss (hunk 1 above)
Comment (KonradKlimczak) [reply src/styles/notification.scss]: Notification still need some work done 😕

**Pull request merged**